libafl/src/corpus/ondisk.rs

```diff
@@ -176,7 +176,7 @@ where
                     break self.dir_path.join(file);
                 }
 
-                file = format!("{}-{}", &file_orig, ctr);
+                file = format!("{}-{ctr}", &file_orig);
```

Comment:
same here

Reply:
Ah, borrows don't work here (?)

Reply:
sounds like we'll fix this again when the lint got updated for borrow in the future 😅

Reply:
No, borrows don't work, and maybe never will. I understood this now, it's just a named parameter that gets filled with a variable of the same name automatically